.github/workflows/script/unitTest/run_unit_test_neuralchat.sh

```diff
@@ -73,6 +73,10 @@ function main() {
     apt-get update
     apt-get install ffmpeg -y
     apt-get install lsof
+    apt-get install libgl1
+    apt-get install -y libgl1-mesa-glx
+    apt-get install -y libgl1-mesa-dev
+    apt-get install libsm6 libxext6 -y
```

Comment:
should this installed in docker images? only for talking photo?

Reply:
The opencv package relies on libGL.so.1, so these packages need to be installed.